docs/internals/howto-release-django.txt

```diff
@@ -229,13 +269,33 @@ __ https://www.djangoproject.com/weblog/2012/nov/27/15-beta-1/
 Actually rolling the release
 ============================
 
-OK, this is the fun part, where we actually push out a release!
+OK, this is the fun part, where we actually push out a release! If you're
+issuing **multiple releases**, repeat these steps for each release.
 
 #. Check `Jenkins`__ is green for the version(s) you're putting out. You
    probably shouldn't issue a release until it's green.
 
    __ https://djangoci.com
 
+#. Cleanup the release notes for this release. Make these changes in ``main``
```

Comment:
I moved this to an earlier step because since these changes should be done in `main` (and then backported), and considering that the following steps occur inside a stable branch, this helps the releaser flow by not require them to move back and forth between `main` and `stable` branches.